src/main/java/net/mcreator/ui/modgui/PlantGUI.java

```diff
@@ -551,6 +562,11 @@ else if (page == 1)
 		plant.generateCondition = generateCondition.getSelectedProcedure();
 		plant.emissiveRendering = emissiveRendering.isSelected();
 
+		useLootTableForDrops.addActionListener(e -> {

```

Comment:
getElementFromGUI does not need this, the implementation in openInEditingMode is fine now.

Move

```
useLootTableForDrops.addActionListener(e -> {
			customDrop.setEnabled(!useLootTableForDrops.isSelected());
			dropAmount.setEnabled(!useLootTableForDrops.isSelected());
		});
```

back to initGUI.

There is no sense for this to be where you put it right now.